.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Question
+about: Doubts about SDMetrics usage
```

Comment:
Change the 'about' to `Ask a general question about SDV usage`. (see [SDV question template](https://github.com/sdv-dev/SDV/blob/master/.github/ISSUE_TEMPLATE/question.md))

Also change `SDMetrics` -> `SDGym` (and below)